package.json

```diff
@@ -28,6 +28,7 @@
     "is-windows": "^1.0.2",
     "js-yaml": "4.1.0",
     "matcher": "^4.0.0",
+    "moment": "^2.29.1",
```

Comment:
Moment is no longer supported. Let me know which of these options seems best:
https://blog.logrocket.com/4-alternatives-to-moment-js-for-internationalizing-dates/

Reply:
Oh dang- I didn't realize 😱  Thanks I'll find a good alternative